packages/teleport/src/Console/DocumentSsh/useSshSession.ts

```diff
@@ -39,74 +43,80 @@ export default function useSshSession(doc: DocumentSsh) {
 
   React.useEffect(() => {
     // initializes tty instances
-    function initTty(session: Session) {
-      const tty = ctx.createTty(session);
-
-      // subscribe to tty events to handle connect/disconnects events
-      tty.on(TermEventEnum.CLOSE, () => ctx.closeTab(doc));
-
-      tty.on(TermEventEnum.CONN_CLOSE, () =>
-        ctx.updateSshDocument(doc.id, { status: 'disconnected' })
-      );
-
-      tty.on('open', () => handleTtyConnect(ctx, session, doc.id));
+    function initTty(session) {
+      tracer.startActiveSpan('initTTY', undefined, context.active(), span => {
```

Comment:
can you leave an inline comment on `undefined`, seems to be a pattern in the backend where if we use values like `true` or `false` we add what it is to be clear and i think it's helpful

```suggestion
      tracer.startActiveSpan('initTTY', undefined /* variable name */, context.active(), span => {
```

Reply:
I've added a comment as requested but I'm not really sure that this is a pattern that's necessary in TS. In JS sure, but in TS the type is clearly defined.